packages/components/src/navigator/stories/index.js

```diff
@@ -18,17 +18,34 @@ export default {
 	component: NavigatorProvider,
 };
 
-function NavigatorButton( { path, isBack = false, ...props } ) {
-	const navigator = useNavigator();
+function NavigatorButton( { path, ...props } ) {
+	const { push } = useNavigator();
+	const dataAttrName = 'data-navigator-focusable-id';
```

Comment:
Should this be just a regular "id"? In general ids are used to navigation/focus so I'm wondering why we need this to be a specific attribute

Reply:
This is the part of this PR where I'm the most open to feedback.

I did think about using an ID, but then I thought that we'd also need to find a way to make sure that we don't have duplicate IDs in the page (in case there was more than one `NavigatorButton` linking to the same `path`) — hence why the data-attribute

Reply:
Good point about the duplication in ID but If we have a duplication in the data attributes, it may also result in the wrong button being focused, so I'm not sure.

Reply:
That's correct. Using data-attributes can't fix the wrong button being focused problem, but at least doesn't introduce duplicate IDs in the page.

To be clear, I'm definitely open to any other approaches as alternatives to data attributes + CSS selectors — I just don't think that using `id`s is a good solution here.

Reply:
`id` is something that would easily be synced to the browser history for Navigator that do require browser history support.

Reply:
I switched to `id` in https://github.com/WordPress/gutenberg/pull/37223/commits/76e88c6782b61c1f37a450f24eb4df4bf03d9635